pkg/controller/group/group_controller.go

```diff
@@ -192,3 +236,57 @@ func (c *Controller) deleteRoleBindings(group *iam1alpha2.Group) error {
 
 	return nil
 }
+
+func (c *Controller) multiClusterSync(group *iam1alpha2.Group) error {
+
+	obj, err := c.federatedGroupLister.Get(group.Name)
+	if obj == nil {
+		return c.createFederatedGroup(group)
+	}
+	if err != nil {
+		klog.Error(err)
+		return err
+	}
+
+	if !reflect.DeepEqual(obj.Spec.Template.Labels, group.Labels) ||
+		!reflect.DeepEqual(obj.Spec.Template.Annotations, group.Annotations) {
+
+		obj.Spec.Template.Annotations = group.Annotations
```

Comment:
The annotations field is [managed by controllers in member clusters](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md#local-value-retention). If the field must be synchronized in multiple clusters, we should use [overrides](https://github.com/kubernetes-sigs/kubefed/blob/master/docs/userguide.md#overrides)

Reply:
The annotations field is not necessary, removed it.